airdrop/cron/repository.py

```diff
@@ -0,0 +1,71 @@
+import pymysql
```

Comment:
repository.py can also be used in the airdrop services. we should keep it in the common folder.